widgets/agents/src/AgentsTable.tsx

```diff
@@ -95,12 +116,15 @@ export default class AgentsTable extends React.Component {
                         }
                     />
                     <DataTable.Column
-                        label="Node"
+                        label={t('columns.node')}
                         show={fieldsToShow.indexOf('Node') >= 0 && !data.nodeId && !data.nodeInstanceId}
                     />
-                    <DataTable.Column label="System" show={fieldsToShow.indexOf('System') >= 0} />
-                    <DataTable.Column label="Version" show={fieldsToShow.indexOf('Version') >= 0} />
-                    <DataTable.Column label="Install Method" show={fieldsToShow.indexOf('Install Method') >= 0} />
+                    <DataTable.Column label={t('columns.system')} show={fieldsToShow.indexOf('System') >= 0} />
+                    <DataTable.Column label={t('columns.version')} show={fieldsToShow.indexOf('Version') >= 0} />
+                    <DataTable.Column
+                        label={t('columns.installMethod')}
+                        show={fieldsToShow.indexOf('Install Method') >= 0}
+                    />
```

Comment:
I'm wondering - can we have:

```suggestion
                    <DataTable.Column label={t('columns.system')} show={fieldsToShow.indexOf(t('columns.system')) >= 0} />
                    <DataTable.Column label={t('columns.version')} show={fieldsToShow.indexOf(t('columns.system')) >= 0} />
                    <DataTable.Column
                        label={t('columns.installMethod')}
                        show={fieldsToShow.indexOf(t('columns.system')) >= 0}
                    />
```
? WDYT? 🤔 

It applies to all places with `fieldsToShow` in modified files.

Reply:
[c89154c](https://github.com/cloudify-cosmo/cloudify-stage/pull/2050/commits/c89154c78b19c154ff6891dad81834aa1b4de21e)